ADMINS.md

```diff
@@ -8,19 +8,27 @@
 
 ## Overview
 
-This document explains who the admins are (see below), what they do in this repo, and how they should be doing it. If you're interested in becoming a maintainer, see [MAINTAINERS](MAINTAINERS.md). If you're interested in contributing, see [CONTRIBUTING](CONTRIBUTING.md).
+This document explains who the admins are (see below), what they do in opensearch project, and how they should be doing it. If you're interested in becoming a maintainer, see [MAINTAINERS](MAINTAINERS.md). If you're interested in contributing, see [CONTRIBUTING](CONTRIBUTING.md).
 
 ## Current Admins
 
```

Comment:
Nit; if you wanted to put a link to the opensearch-project team here, this would be a good spot for it.

"Admin are managed through the OpenSearch-Project organization on team named Admin, members of the organization can see the admin list at [link]"

Reply:
i agree, if the link is not public accessible, it will cause confusion. I hope there is more open and transparent and elegant way to handle this with github.